### PR TITLE
added: firebase, service

### PR DESCRIPTION
**Pull request by VeiveHata**
basic service has been created; connecting with firebase

### CODE REVIEW

File: src/app/components/questions/questions.component.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+@Component({
+  selector: 'app-questions',
+  templateUrl: './questions.component.html',
+  styleUrls: ['./questions.component.css']
+})
+export class QuestionsComponent implements OnInit {
+
+	questions: any[] = [];
+
+	constructor(private questionServis: QuestionService) { }
```
Comment (FrontendMentor) [inline src/app/components/questions/questions.component.ts]: It seems like there is a misprint. It should read `questionService`

File: src/app/components/questions/questions.component.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    ngOnInit() {
+  		this.questionServis.getQuestions().subscribe(questions => {
```
Comment (FrontendMentor) [inline src/app/components/questions/questions.component.ts]: Please, set type for `questions `

File: src/app/services/question.service.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+	private basePath: string = "/questions";
+
+	question: AngularFireList<Question> = null;
+	questions: Observable<Question[]> = null;
+
+    constructor(private db: AngularFireDatabase) {
+    	this.questions = db.list('questions').valueChanges();    	
+    }
+
+    getQuestions() {
```
Comment (FrontendMentor) [inline src/app/services/question.service.ts]: Please, add return type for this method

File: src/app/services/question.service.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+@Injectable()
+export class QuestionService {
+
+	private basePath: string = "/questions";
+
+	question: AngularFireList<Question> = null;
+	questions: Observable<Question[]> = null;
+
+    constructor(private db: AngularFireDatabase) {
+    	this.questions = db.list('questions').snapshotChanges().map(changes => {
```
Comment (FrontendMentor) [inline src/app/services/question.service.ts]: set type for `questions`

File: src/app/services/question.service.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+      		return changes.map(c => ({ key: c.payload.key, ...c.payload.val() }));
```
Comment (FrontendMentor) [inline src/app/services/question.service.ts]: please, rename `c` and add type

File: src/app/services/question.service.ts (hunk 4 above)
Comment (VeiveHata) [reply src/app/services/question.service.ts]: the same as questionUnit type

**Comment by FrontendMentor** [comment]
This pull request is part of #2

**Pull request closed without merging**